README.md

```diff
@@ -123,6 +123,42 @@ For details on environment configuration, including optional variables, see
 
 The easiest way to develop MBTA dotcom is to use Docker Compose.
 
+### Installing Rancher Desktop
+
+We have tested this setup using [Rancher Desktop](https://rancherdesktop.io) on Mac.
+
+Install Rancher Desktop and then download the [docker-credential-osxkeychain](https://github.com/docker/docker-credential-helpers/releases) binary.
+Add it to your `$PATH`.
+
+Install Docker and Docker Compose:
+
+```
+brew install docker docker-compose
+```
+
+Make sure your `~/.docker/config.json` looks like the following:
+
+```json
+{
+  "auths": {},
+  "cliPluginsExtraDirs": [
+    "/opt/homebrew/lib/docker/cli-plugins"
+  ],
+  "credsStore": "osxkeychain",
+  "currentContext": "rancher-desktop"
+}
+```
+
+Login to Docker:
+
+```
+docker login
+```
```

Comment:
How come the `docker login` step is necessary?

Reply:
I guess it's not necessary for Dotcom since we use a different registry. But, it is part of setting up your local Docker if you're going to be using it for anything else.